editor/src/components/inspector/fill-hug-fixed-control.tsx

```diff
@@ -298,7 +303,7 @@ export const FillHugFixedControl = React.memo<FillHugFixedControlProps>((props)
     >
       <PopupList
         value={optionalMap(selectOption, widthCurrentValue?.type) ?? undefined}
-        options={optionsRef.current}
+        options={options}
         onSubmitValue={onSubmitWidth}
         controlStyles={controlStylesRef.current}
```

Comment:
while you are here, I think this is also wrong!

Reply:
fixed in [6ed5ddb](https://github.com/concrete-utopia/utopia/pull/3270/commits/6ed5ddb5b563de88bbacd03af1996e5d02ceed4f)